projects/v2/src/app/components/home/home.component.ts

```diff
@@ -31,10 +32,17 @@ export class HomeComponent implements OnInit, AfterViewInit {
   faEnvelope = faEnvelope;
 
   copyrightYear = new Date().getFullYear();
+  masterSheetLink:string;
```

Comment:
always have a space between the property and it's type. ie `masterSheetLink: string;`